README.md

```diff
@@ -12,15 +12,101 @@ import * as minFraud from '@maxmind/minfraud-api-node';
 // const minFraud = require('@maxmind/minfraud-api-node');
 
 // client is reusable
-const client = new minFraud.WebServiceClient("1234", "LICENSEKEY");
+const client = new minFraud.Client("1234", "LICENSEKEY");
 
 let transaction;
 
 try {
   transaction = new minFraud.Transaction({
+    // device is required
     device: new minFraud.Device({
       ipAddress: "8.8.8.8",
     }),
+    event: new minFraud.Event({
+      shopId: 'shop',
+      time: new Date(Date.now()),
+      transactionId: 'txn1234',
+      type: minFraud.Constants.EventType.PayoutChange,
+    }),
+    account: new minFraud.Account({
+      userId: 'user123',
+      username: 'userperson',
+    }),
+    email: new minFraud.Email({
+      address: 'foo@bar.com',
+      domain: 'bar.com',
+    }),
+    billing: new minFraud.Billing({
+      address: '123 Robot Ave.',
+      address2: 'Suite 10011',
+      city: 'Waltham',
+      company: 'Robots Inc.',
+      country: 'US',
+      firstName: 'Robot',
+      lastName: 'Bar',
+      phoneCountryCode: '1',
+      phoneNumber: '321-321-3211',
+      postal: '12345',
+      region: 'MA',
+    }),
+    shipping: new minFraud.Shipping({
+      address: '123 Robot Ave.',
+      address2: 'Suite 10011',
+      city: 'Waltham',
+      company: 'Robots Inc.',
+      country: 'US',
+      deliverySpeed: minFraud.Constants.DeliverySpeed.Expedited,
+      firstName: 'Robot',
+      lastName: 'Bar',
+      phoneCountryCode: '1',
+      phoneNumber: '321-321-3211',
+      postal: '12345',
+      region: 'MA',
+    }),
+    payment: new minFraud.Payment({
+      declineCode: 'A',
+      processor: minFraud.Constants.Processor.ConceptPayments,
```

Comment:
Maybe drop the `Constants` so that it's `minFraud.Processor.ConeptPayments`?

Reply:
I think it's fine to leave it under `Constants`. I'm thinking that for an outsider that is unfamiliar with our code, they might have a easier time understanding what `minFraud.Constants.Processor.ConceptPayments` is at first glance because it provides a tiny bit more context.